accord-core/src/test/java/accord/verify/StrictSerializabilityVerifier.java

```diff
@@ -28,49 +28,53 @@
 import java.util.TreeMap;
 import java.util.TreeSet;
 import java.util.function.Consumer;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
-import accord.utils.Invariants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Nomenclature:
- *  register: the values associated with a given key
- *  step: a logical point in the causal sequence of events for a register
- *  step index: the index of a step; since we observe a strictly growing sequence this translates directly to
- *              the length of an observed sequence for a key. This imposes a total causal order for a given register
- *              (sequences [], [1], [1,2] have step indexes of 0, 1 and 2 respectively, with 2 necessarily happening after 1 and 0)
- *  predecessor: a step for a referent key that must occur before the referring step and key.
- *               two kinds: 1) those values for keys [B,C..) at step index i for a read of key A, precede key A's step index i +1
- *                          2) those values for keys [B,C..) at step index i for a write of key A, precede key A's step index i
- *  max predecessor: the maximum predecessor that may be reached via any predecessor relation
- *
+ * register: the values associated with a given key
+ * step: a logical point in the causal sequence of events for a register
+ * step index: the index of a step; since we observe a strictly growing sequence this translates directly to
```

Comment:
why removing the space at front? it works as indentation under the Nomenclature heading?